### ISSUE TITLE
[Feature request] Remove custom scrollbar styling for sidebar

### ISSUE THREAD

**Issue by chris-dura** [vuepress/vuepress-next#674]
### Description

Custom scrollbar CSS styling is undesirable for accessibility, and some users prefer the native browser defaults. However, once the styles (eg, `::-webkit-scrollbar-*`) are applied, they cannot be reset to the browser defaults, only overridden to emulate the browser defaults (which are obviously different for each browser).

### Proposed Solution

Remove the `::-webkite-scrollbar-*` styles from the core default theme styles, so users can keep the native browser defaults if they want. Or, provide the ability for users to "turn on" the default-theme's custom scrollbar styling via Config or Plugin, so that `::-webkit-scrollbar-*` CSS is only output for users that opt-in to it.
